src/main/java/game/domain/board/Board.java

```diff
@@ -0,0 +1,86 @@
+package game.domain.board;
+
+import static game.domain.piece.Team.CHO;
+import static game.domain.piece.Team.DEFAULT;
+import static game.domain.piece.Team.HAN;
+
+import game.domain.piece.Cannon;
+import game.domain.piece.Chariot;
+import game.domain.piece.DefaultPiece;
+import game.domain.piece.King;
+import game.domain.piece.Pawn;
+import game.domain.piece.Piece;
+import game.domain.piece.Scholar;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Board {
+
+    private final Map<BoardLocation, Piece> pieces;
+
+    public Board(Map<BoardLocation, Piece> pieces) {
+        this.pieces = pieces;
+    }
+
+    public Piece getByLocationOrThrow(BoardLocation current) {
+        if (pieces.containsKey(current)) {
+            return pieces.get(current);
+        }
+        throw new IllegalArgumentException("[ERROR] 해당 위치에 기물이 없습니다.");
+    }
+
+    public Piece getByLocationOrDefault(BoardLocation current) {
+        if (pieces.containsKey(current)) {
+            return pieces.get(current);
+        }
+        return new DefaultPiece(DEFAULT);
+    }
```

Comment:
아래와 같이 수정도 가능할 것 같아요.

```suggestion
    public Piece getByLocationOrDefault(BoardLocation current) {
        return pieces.getOrDefault(current, new DefaultPiece(DEFAULT));
    }
```